automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/billing/BillingSpec.scala

```diff
@@ -21,6 +22,8 @@ import org.scalatest.{FreeSpec, Matchers}
 class BillingSpec extends FreeSpec with WebBrowserSpec with UserFixtures with CleanUp
   with Matchers with WorkspaceFixtures with BillingFixtures with LazyLogging {
 
+  implicit override val patienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(500, Millis)))
```

Comment:
`shouldEqual` `shouldBe` and other types of assertion checks are wrapped in `eventually` to handle slow ui refresh.